app/views/quotes/_quote.html.erb

```diff
@@ -12,19 +12,25 @@
     <p class="text-gray-700 text-base mb-4">
       <%= quote.content %>
     </p>
-    <p class="text-gray-500 text-base mb-4">
-      <%= quote.likes.count %> like(s)
-    </p>
-    <% if user_signed_in? %>
-      <% if quote.liked_by?(current_user)  %>
-        <%= button_to likes_path(quote_id: quote.id, page: params['page']), method: :delete do %>
-          <i class="fa-solid fa-heart text-pink-600 text-xl"></i>
-        <% end %>
-      <% else %>
-        <%= button_to likes_path(quote_id: quote.id, page: params['page']) do %> 
-          <i class="fa-regular fa-heart text-pink-600 text-xl"></i>
+    <% if quote.state == 'approved' %>
+      <p class="text-gray-500 text-base mb-4">
+        <%= quote.likes.count %> like(s)
+      </p>
+      <% if user_signed_in? %>
+        <% if quote.liked_by?(current_user)  %>
+          <%= button_to likes_path(quote_id: quote.id, page: params['page']), method: :delete do %>
+            <i class="fa-solid fa-heart text-pink-600 text-xl"></i>
+          <% end %>
+        <% else %>
+          <%= button_to likes_path(quote_id: quote.id, page: params['page']) do %> 
+            <i class="fa-regular fa-heart text-pink-600 text-xl"></i>
+          <% end %>
         <% end %>
       <% end %>
+    <% else %>
+      <p class="text-red-700 text-base mb-4">
+        <%= t('.pending_quote') %>
+      </p>
```

Comment:
why this is present in 2 PRs ?
here as well https://github.com/yuliatokaryk/quotes/pull/47/files